emoji/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
   testImplementation rootProject.ext.testing.assertJ
   testImplementation rootProject.ext.testing.robolectric
   testImplementation rootProject.ext.testing.privateConstructor
+  androidTestAnnotationProcessor deps.support.test.autoService
```

Comment:
What do we need this for?

Reply:
One of the lints complained about it missing.